### PR TITLE
fix(publish): chain id setup

### PR DESCRIPTION
**Pull request by mjlescano**
This PR fixes two bugs:
1. When running `cannon publish` including `clone` steps, it was wrongly publishing the subpackages with the `13370` chain instead of the target chain, that is addressed here: https://github.com/usecannon/cannon/pull/1408/files#diff-756d4c0d0bd036727ea6922554c95272acc0b2e94bd8311308f1a69574d94f82R98
2. When running `cannon build`, the imports of clone steps were having always the chainId `13370`, this is fixed here: https://github.com/usecannon/cannon/pull/1408/files#diff-950232bdec873b2ee3a9f82132b20b978b8ae9fcf3be3a475e613efee95046f6R256

### CODE REVIEW

**Review by FuzzB0t — APPROVED** [review]

File: .nx/workspace-data/d/server-process.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"processId":63247}
```
Comment (dbeal-eth) [inline .nx/workspace-data/d/server-process.json:1]: remove

File: .nx/workspace-data/lockfile.hash [hunk 2]
```diff
@@ -1 +1 @@
-15584140240147292807
\ No newline at end of file
+11580160284360395692
```
Comment (dbeal-eth) [inline .nx/workspace-data/lockfile.hash:1]: remove?

File: packages/builder/src/steps/clone.ts [hunk 3]
```diff
@@ -253,7 +253,7 @@ const cloneSpec = {
       state: builtState,
       meta: deployInfo.meta,
       status: partialDeploy ? 'partial' : 'complete',
-      chainId,
+      chainId: runtime.chainId,
```
Comment (dbeal-eth) [inline packages/builder/src/steps/clone.ts:256]: this is wrong. `clone` steps should be pulling from chain ID 13370 by default. This is as intended. Please undo.

**Review by dbeal-eth — CHANGES_REQUESTED** [review]

**Comment by dbeal-eth** [comment]
fix confirmed, going to merge this as is.

**Review by dbeal-eth — APPROVED** [review]

**Pull request merged**